kwonhayeon/SpringPrac/src/main/java/com/likelion/springprac/domain/User.java

```diff
@@ -0,0 +1,15 @@
+package com.likelion.springprac.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data // getter, setter, toString 등 자동 생성
+@AllArgsConstructor // 모든 필드를 받는 생성자
+@NoArgsConstructor // 기본 생성자
+
+public class User {
+    private Long id;
```

Comment:
- `@Id`을 사용하여 해당 필드가 기본키임을 지정해주어야해요.
- `@GeneratedValue(strategy = GenerationType.IDENTITY) `를 사용하면 다른 코드를 작성해 값을 올릴 필요 없이 JPA가 알아서 관리해줘요!

```suggestion
    @Id 
    @GeneratedValue(strategy = GenerationType.IDENTITY) 
    private Long id;
```